easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart

```diff
@@ -118,105 +118,113 @@ class NFTsListTile extends StatelessWidget {
-                      return Container(
-                        margin: EdgeInsets.symmetric(vertical: 5.h, horizontal: 3.w),
-                        decoration: BoxDecoration(
-                          color: EaselAppTheme.kDarkGrey02,
-                          boxShadow: [
-                            BoxShadow(
-                              color: Colors.black.withOpacity(0.3),
-                              offset: const Offset(0.0, 1.0),
-                              blurRadius: 4.0,
-                            ),
-                          ],
+            ),
+          )
+        else
+          Padding(
+            padding: EdgeInsets.symmetric(horizontal: 20.w),
+            child: getPublishedCard(context: context),
+          ),
+        if (publishedNFT.assetType.toAssetTypeEnum() != AssetType.ThreeD)
+          IgnorePointer(
+            child: SizedBox(
+              height: 85.0.h,
+              width: double.infinity,
+              child: CachedNetworkImage(
+                imageUrl: publishedNFT.assetType.toAssetTypeEnum() == AssetType.Image ? publishedNFT.url.changeDomain() : publishedNFT.thumbnailUrl.changeDomain(),
+                fit: BoxFit.fill,
+                color: Colors.transparent,
+                colorBlendMode: BlendMode.clear,
+                placeholder: (context, _) {
+                  return Container(
+                    margin: EdgeInsets.symmetric(vertical: 5.h, horizontal: 3.w),
+                    decoration: BoxDecoration(
+                      color: EaselAppTheme.kDarkGrey02,
+                      boxShadow: [
+                        BoxShadow(
+                          color: Colors.black.withOpacity(0.3),
+                          offset: const Offset(0.0, 1.0),
+                          blurRadius: 4.0,
                         ),
-                        child: Padding(
-                          padding: EdgeInsets.symmetric(horizontal: 15.w, vertical: 15.h),
-                          child: Row(
-                            children: [
-                              Container(
-                                width: 50.0.w,
-                                height: 50.0.w,
-                                decoration: const BoxDecoration(
+                      ],
+                    ),
+                    child: Padding(
+                      padding: EdgeInsets.symmetric(horizontal: 15.w, vertical: 15.h),
+                      child: Row(
+                        children: [
+                          Container(
+                            width: 50.0.w,
+                            height: 50.0.w,
+                            decoration: const BoxDecoration(
+                              color: EaselAppTheme.kLightGrey03,
+                            ),
+                          ),
+                          SizedBox(
+                            width: 10.w,
+                          ),
+                          Expanded(
```

Comment:
What this is for ? I see only empty list.

Reply:
it is a place holder for cacheImages.

Reply:
strange is there any better way?